lib/rubygems/commands/verify_extend.rb

```diff
@@ -25,13 +26,32 @@
   Gem::Sigstore.options[:verify] = value
 end
 
-Gem.pre_install do |installer|
-  begin
-    if (Gem::Sigstore.options[:verify])
-      puts "verify called"
+i.add_option('--rekor-host HOST', 'Rekor host') do |value, options|
+  options[:host] = value
+end
+
+class Gem::Commands::InstallCommand
+  alias_method :original_execute, :execute
+  def execute
```

Comment:
Any particular reasons why we're not leveraging the `pre_install` hook for plugins?  I would only reopen the command class and redefine its entry method if necessary.

Reply:
I was checking if it makes a difference of the flag is implemented.